src/app/stats/client.tsx

```diff
@@ -56,15 +58,15 @@ export function Podium({ scores }: PodiumProps) {
     return <div style={styles.podiumContainer}>
         <div style={styles.podiumBase}>
             <AnimatedPlacer username={placers[1]?.user_name} score={placers[1]?.score} pos={1} style={placerTrails[1]} />
-            <animated.div style={{ ...styles.podium, ...platformTrails[1], borderTopRightRadius: 0 }} />
+            <animated.div style={{ ...styles.podium, ...platformTrails[1], borderTopRightRadius: 0 }} suppressHydrationWarning={true} />
```

Comment:
Hydration warnings were being triggered by react-spring converting css `var(--color-transparent)` to the value of the css variable on the first render.